.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.1.0
+    rev: v4.4.0
     hooks:
       - id: check-merge-conflict
       - id: trailing-whitespace
 
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.45.2
+    rev: v1.51.2
```

Comment:
I was getting `golangci-lint` errors locally until I updated this.  May have been something particular to me, but figured it didn't hurt to upgrade to newer linters.